.gitignore

```diff
@@ -109,6 +109,7 @@ scripts/guid_blacklist.txt
 
 # Generated json
 website/static/js/_allLogTexts.json
+website/static/js/_anonymousLogTexts.json
```

Comment:
There's a /website/static/built/ folder that is always hidden; might make sense to use in this case.